src/Core/Domain/Customer/Group/Command/AddCustomerGroupCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace PrestaShop\PrestaShop\Core\Domain\Customer\Group\Command;
+
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\ValueObject\GroupName;
+
+class AddCustomerGroupCommand
+{
+    /** @var string[] */
+    private $localizedNames;
+    /** @var float */
+    private $reduction;
+    /** @var int */
+    private $priceDisplayMethod;
+    /** @var bool */
+    private $showPrice;
```

Comment:
again cs fixer should complain about this ideally.
```suggestion
     /**
     * @var string[]
     */
    private $localizedNames;
    
     /**
     * @var float
     */
    private $reduction;
    
     /**
     * @var int
     */
    private $priceDisplayMethod;
    
     /**
     * @var bool
     */
    private $showPrice;
```

Reply:
Indeed please keep the EOL between each field for the convention, we really need to integrate this to CSFixer or PHPStan

Reply:
Fixed